packages/protocol/contracts/governance/ReleaseGold.sol

```diff
@@ -272,6 +277,20 @@ contract ReleaseGold is UsingRegistry, ReentrancyGuard, IReleaseGold, Initializa
     emit LiquidityProvisionSet(beneficiary);
   }
 
+  /**
+   * @notice Controls if the contract can be expired.
+   *         This would be called by beneficiarys who would like to validate with this contract
+   *         without worrying about future expiration.
+   */
+  function setCanExpire(bool _canExpire) external onlyBeneficiary {
```

Comment:
nit: do you need the `_`?

Reply:
to keep the same name, which I think is clean, yes